google-cloud-clients/google-cloud-resourcemanager/src/main/java/com/google/cloud/resourcemanager/ProjectInfo.java

```diff
@@ -106,50 +106,6 @@ public static State valueOf(String constant) {
     }
   }
 
-  static class ResourceId implements Serializable {
```

Comment:
Why we need a separate class to expose `ResourceId` as public? I mean a customer could easily access `ProjectInfo.ResourceId` when you make it `public static class ResourceId implements Serializable{ ... }`

Reply:
yes but while building builder in existing code you need to pass ResourceId model of api services instead of static ResourceId,

Reply:
>  ResourceId model of api services instead of static ResourceId

If that is the case then making it a separate class would not solve it... Can you please elaborate more?